src/libCrypto/EthCrypto.cpp

```diff
@@ -334,3 +334,31 @@ bytes GetOriginalHash(TransactionCoreInfo const& info, uint64_t chainId) {
 
   return bytes{&signingHash.bytes[0], &signingHash.bytes[32]};
 }
+
+bytes ToEVM(bytes const& in) {
+  if (in.empty()) {
+    return in;
+  }
+
+  std::string prefixedEvm{"EVM"};
+  prefixedEvm += DataConversion::Uint8VecToHexStrRet(in);
+  bytes ret;
+
+  for (const auto& a : prefixedEvm) {
```

Comment:
Nitpick: it's a good candidate for `std::copy()` with `std::back_inserter`.

Reply:
👍